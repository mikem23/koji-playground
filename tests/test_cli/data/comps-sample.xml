<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
<comps>
  <group>
    <id>additional-devel</id>
    <name>Additional Development</name>
    <description>Additional development headers and libraries for developing applications</description>
    <default>false</default>
    <uservisible>false</uservisible>
    <biarchonly>true</biarchonly>
    <langonly>fr</langonly>
    <packagelist>
      <packagereq type="default">alsa-lib-devel</packagereq>
      <packagereq type="default">audit-libs-devel</packagereq>
      <packagereq type="default">binutils-devel</packagereq>
      <packagereq type="default">boost-devel</packagereq>
      <packagereq type="default">bzip2-devel</packagereq>
      <packagereq type="default">cyrus-sasl-devel</packagereq>
    </packagelist>
  </group>
  <group>
    <id>backup-client</id>
    <name>Backup Client</name>
    <description>Client tools for connecting to a backup server and doing backups.</description>
    <default>true</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory">amanda-client</packagereq>
      <packagereq type="optional">bacula-client</packagereq>
    </packagelist>
  </group>
  <group>
    <id>backup-server</id>
    <name>Backup Server</name>
    <description>Software to centralize your infrastructure's backups.</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory">amanda-server</packagereq>
      <packagereq type="optional">mt-st</packagereq>
      <packagereq type="optional">mtx</packagereq>
    </packagelist>
  </group>
  <group>
    <id>ansible-node</id>
    <name>Ansible node</name>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory">python2-dnf</packagereq>
      <packagereq type="conditional" requires="selinux-policy">libselinux-python</packagereq>
    </packagelist>
  </group>
  <group>
    <id>d-development</id>
    <name>D Development Tools and Libraries</name>
    <description>These include development tools and libraries such as ldc, and geany-tag.</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory" basearchonly="true">ldc</packagereq>
      <packagereq type="mandatory" basearchonly="true">ldc-druntime</packagereq>
      <packagereq type="mandatory" basearchonly="true">ldc-druntime-devel</packagereq>
      <packagereq type="mandatory" basearchonly="true">ldc-phobos-devel</packagereq>
      <packagereq type="mandatory">make</packagereq>
      <packagereq type="mandatory">pkgconfig</packagereq>
      <packagereq type="default">ctags</packagereq>
      <packagereq type="default">indent</packagereq>
      <packagereq type="optional">astyle</packagereq>
      <packagereq type="optional">cmake</packagereq>
      <packagereq type="optional">derelict-devel</packagereq>
      <packagereq type="optional">geany</packagereq>
      <packagereq type="optional">gl3n-devel</packagereq>
      <packagereq type="optional">insight</packagereq>
      <packagereq type="optional">nemiver</packagereq>
      <packagereq type="optional">uncrustify</packagereq>
    </packagelist>
  </group>
  <group>
    <id>empty-group-1</id>
    <name>empty group 1</name>
    <description>empty group 1 desc</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist/>
  </group>
  <group>
    <id>empty-group-2</id>
    <name>empty group 2</name>
    <description>empty group 2 desc</description>
    <default>false</default>
    <uservisible>true</uservisible>
  </group>
    <group>
    <id>unknown-group</id>
    <name>unknown group</name>
    <description>unknown group desc</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="unknown">unknown</packagereq>
      <packagereq type="what">unknown2</packagereq>
    </packagelist>
  </group>
</comps>
