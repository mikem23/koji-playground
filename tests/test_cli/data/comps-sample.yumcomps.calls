[call.groupListAdd('tag', 'additional-devel', description='Additional development headers and libraries for developing applications', display_name='Additional Development', force=None, is_default=False, langonly='fr', uservisible=False),
 call.groupPackageListAdd('tag', 'additional-devel', 'binutils-devel', force=None, type='default'),
 call.groupPackageListAdd('tag', 'additional-devel', 'boost-devel', force=None, type='default'),
 call.groupPackageListAdd('tag', 'additional-devel', 'bzip2-devel', force=None, type='default'),
 call.groupPackageListAdd('tag', 'additional-devel', 'cyrus-sasl-devel', force=None, type='default'),
 call.groupPackageListAdd('tag', 'additional-devel', 'audit-libs-devel', force=None, type='default'),
 call.groupPackageListAdd('tag', 'additional-devel', 'alsa-lib-devel', force=None, type='default'),
 call.groupListAdd('tag', 'ansible-node', description='', display_name='Ansible node', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', 'ansible-node', 'python2-dnf', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'ansible-node', 'libselinux-python', force=None, requires='selinux-policy', type='conditional'),
 call.groupListAdd('tag', 'backup-client', description='Client tools for connecting to a backup server and doing backups.', display_name='Backup Client', force=None, is_default=True, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', 'backup-client', 'amanda-client', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'backup-client', 'bacula-client', force=None, type='optional'),
 call.groupListAdd('tag', 'backup-server', description="Software to centralize your infrastructure's backups.", display_name='Backup Server', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', 'backup-server', 'amanda-server', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'backup-server', 'mtx', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'backup-server', 'mt-st', force=None, type='optional'),
 call.groupListAdd('tag', 'd-development', description='These include development tools and libraries such as ldc, and geany-tag.', display_name='D Development Tools and Libraries', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', 'd-development', 'ldc', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'd-development', 'ldc-druntime', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'd-development', 'make', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'd-development', 'pkgconfig', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'd-development', 'ldc-phobos-devel', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'd-development', 'ldc-druntime-devel', force=None, type='mandatory'),
 call.groupPackageListAdd('tag', 'd-development', 'ctags', force=None, type='default'),
 call.groupPackageListAdd('tag', 'd-development', 'indent', force=None, type='default'),
 call.groupPackageListAdd('tag', 'd-development', 'geany', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'cmake', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'derelict-devel', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'insight', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'nemiver', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'gl3n-devel', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'astyle', force=None, type='optional'),
 call.groupPackageListAdd('tag', 'd-development', 'uncrustify', force=None, type='optional'),
 call.groupListAdd('tag', 'empty-group-1', description='empty group 1 desc', display_name='empty group 1', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupListAdd('tag', 'empty-group-2', description='empty group 2 desc', display_name='empty group 2', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupListAdd('tag', 'unknown-group', description='unknown group desc', display_name='unknown group', force=None, is_default=False, langonly=None, uservisible=True)]
