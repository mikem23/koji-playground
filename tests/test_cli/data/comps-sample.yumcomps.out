WARN: yum.comps does not support the biarchonly of group and basearchonly of package
Group: additional-devel (Additional Development)
  Package: binutils-devel: {'type': 'default'}
  Package: boost-devel: {'type': 'default'}
  Package: bzip2-devel: {'type': 'default'}
  Package: cyrus-sasl-devel: {'type': 'default'}
  Package: audit-libs-devel: {'type': 'default'}
  Package: alsa-lib-devel: {'type': 'default'}
Group: ansible-node (Ansible node)
  Package: python2-dnf: {'type': 'mandatory'}
  Package: libselinux-python: {'requires': 'selinux-policy', 'type': 'conditional'}
Group: backup-client (Backup Client)
  Package: amanda-client: {'type': 'mandatory'}
  Package: bacula-client: {'type': 'optional'}
Group: backup-server (Backup Server)
  Package: amanda-server: {'type': 'mandatory'}
  Package: mtx: {'type': 'optional'}
  Package: mt-st: {'type': 'optional'}
Group: d-development (D Development Tools and Libraries)
  Package: ldc: {'type': 'mandatory'}
  Package: ldc-druntime: {'type': 'mandatory'}
  Package: make: {'type': 'mandatory'}
  Package: pkgconfig: {'type': 'mandatory'}
  Package: ldc-phobos-devel: {'type': 'mandatory'}
  Package: ldc-druntime-devel: {'type': 'mandatory'}
  Package: ctags: {'type': 'default'}
  Package: indent: {'type': 'default'}
  Package: geany: {'type': 'optional'}
  Package: cmake: {'type': 'optional'}
  Package: derelict-devel: {'type': 'optional'}
  Package: insight: {'type': 'optional'}
  Package: nemiver: {'type': 'optional'}
  Package: gl3n-devel: {'type': 'optional'}
  Package: astyle: {'type': 'optional'}
  Package: uncrustify: {'type': 'optional'}
Group: empty-group-1 (empty group 1)
Group: empty-group-2 (empty group 2)
Group: unknown-group (unknown group)
